test/HiGlassComponentTest.js

```diff
@@ -670,7 +669,7 @@ describe('Simple HiGlassComponent', () => {
 
       hgc.instance().setState({ views });
 
-      const selection = select(ReactDOM.findDOMNode(hgc.instance()))
+      const selection = select(ReactDOM.findDOMNode(hgc.instance())) // eslint-disable-line
```

Comment:
Please only disable a specific rule to make it clear. I know this can be annoying but the current solution might hide errors.

Reply:
I suppose this is complaining about `findDOMNode`?

But in general I agree with Fritz. If possible it would be more clear if one could see why linting was disabled for a particular line.